### PR TITLE
Peterhart/lending data

### PR DESCRIPTION
**Pull request by peter-hart-ri**
Part of RIME-9808

### CODE REVIEW

File: tabular/lending/configs/stress_test_config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "data_info": {
+        "ref_path": "data/ref_with_preds.csv",
+        "eval_path": "data/eval_with_preds_with_timestamps.csv",
```
Comment (jerryjliu) [inline tabular/lending/configs/stress_test_config.json]: stress test config should not have timestamps in eval data

File: tabular/lending/model_urls.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+https://www.dropbox.com/s/5ib4po1jeyrjtd5/cat_features.pkl?dl=01,cat_features.pkl
```
Comment (jerryjliu) [inline tabular/lending/model_urls.csv]: nit: dl=1

File: tabular/lending/configs/incremental_config [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "eval_path": rime_testing_path,
```
Comment (jerryjliu) [inline tabular/lending/configs/incremental_config]: are we checking in this file? lots of issues with it atm, maybe we shouldn't for now

Comment (peter-hart-ri) [reply tabular/lending/configs/incremental_config]: I don't need either config for the notebook to work so I'm happy to leave this out

**Review by jerryjliu — APPROVED** [review]

**Pull request merged**